test/main.js

```diff
@@ -185,9 +187,11 @@ const createMocha = (driver, testCase) => {
     //As we are running a 'single' test as test/specs/chrome.js, we will only be able to report a single error
     //i.e. if we have 3 failures...BS will only log the first one.
     if (isRemoteBrowser && currentTestState === 'failed') {
+      browsersFailures[browserName] = +1
       browserStackFailures += 1
```

Comment:
Minor: Why not just write `browsersFailures[browserName]` in the same way as `browserStackFailures` since the are both incrementing their respective variable value by 1?

Reply:
I've removed this and just keeping a single way now.